README.md

```diff
@@ -67,7 +67,7 @@ sudo yum install frr frr-pythontools
 
 ### Deploy Citrix ADC CPX
 #### Deploy Citrix ADC CPX using docker-compose
-The following `docker-compose.yml` deploys a Citrix ADC CPX container :
+The following `docker-compose.yml` deploys a pair of Citrix ADC CPX container :
```

Comment:
Both CPX instances have to be deployed on 2 different physical or virtual hosts for effective resiliency. IP anycast is not needed if they are deployed on the same host as they are deployed on the same L3 subnet : unicast VIP in the L3 subnet is enough.

Anycast is required when CPX HA pair is deployed on physical or virtual hosts being on disjoint L3 subnets.

We should reflect that in the documentation and explain that the `docker-compose.yml` has to be played twice with the `HOST` env variable being configured with the physical / virtual docker host public IP address. This public IP address is used to connect to HA peer TCP/3008 RPC port.

Support use of shell commands at env variable docker-compose PR : https://github.com/docker/compose/pull/7843